src/Raven.Server/Documents/PeriodicBackup/BackupUploader.cs

```diff
@@ -226,14 +230,17 @@ private string CombinePathAndKey(string path)
                         }
                     }
                 }
-                catch (OperationCanceledException e)
-                {
-                    // shutting down
-                    localUploadStatus.Exception = e.ToString();
-                    _exceptions.Add(e);
-                }
                 catch (Exception e)
                 {
+                    var extracted = e.ExtractSingleInnerException();
+                    if (extracted is OperationCanceledException)
```

Comment:
aren't we hiding potential exception from HttpClient here? Shouldn't we check that databaseShutdownToken is cancelled?

Reply:
done